src/main/java/net/minecraftforge/common/DimensionManager.java

```diff
@@ -412,42 +411,37 @@ public static void unloadWorlds(Hashtable<Integer, long[]> worldTickTimes)
      * block of free ids. Always call for each individual ID you wish to get.
      * @return the next free dimension ID
      */
-    public static int getNextFreeDimId() {
+    public static int getNextFreeDimId()
+    {
         int next = 0;
```

Comment:
Don't need to start at zero every time, keep a static var holding the last returned value. A from-scratch search will need to be done every launch which avoids any issues with uninstalled mods.